build/functions.j2

```diff
@@ -32,7 +32,7 @@ function process_command_exit_status() {
 {% macro function(function, file_extensions, security_tools, allow_filter, version_argument, validations) %}
 function {{ function }}() {
   arguments=("${@}")
-  easy_infra_{{ function | replace("-", "_") }}_tools=({% for security_tool in security_tools %}{{ security_tool | replace("-", "_") }}{% if not loop.last %} {% endif %}{% endfor %})
+  easy_infra_{{ function | replace("-", "_") }}_security_tools=({% for security_tool in security_tools %}{{ security_tool | replace("-", "_") }}{% if not loop.last %} {% endif %}{% endfor %})
```

Comment:
This is for clarity; we now have consistenc throughout the project on what is a `tool` vs a `security_tool` vs a `helper` and a `tag` vs `version`, etc.  More details in the original comment of this PR